### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by arb**
Removed `objHash`
Added dependencies badge 
Used JS .travis config because I dropped `objHash`

This is a breaking change because I stopped use `Object.defineProperty` because it's slow and if the developer want's to be crazy and do weird stuff, that's their issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # stand-in
 Method replacement and testing utility
 
-[![build status](https://travis-ci.org/continuationlabs/stand-in.svg?branch=v2.0.1)](https://travis-ci.org/continuationlabs/stand-in)
+[![Build Status](https://travis-ci.org/continuationlabs/stand-in.svg?branch=master)](https://travis-ci.org/continuationlabs/stand-in)
 [![npm version](https://img.shields.io/npm/v/stand-in.svg)](https://www.npmjs.org/package/stand-in)
+[![Dependencies](https://img.shields.io/david/continuationlabs/stand-in.svg)](https://github.com/continuationlabs/stand-in)
```
Comment (cjihrig) [inline README.md:6]: Nice

File: README.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+
+[![js-semistandard-style](https://cdn.rawgit.com/flet/semistandard/master/badge.svg)](https://github.com/Flet/semistandard)
```
Comment (cjihrig) [inline README.md:8]: Are we going to stick with semistandard?

File: lib/index.js [hunk 3]
```diff
@@ -1,40 +1,51 @@
...
-
-  Object.defineProperty(result, 'original', {
-    value: obj[method].bind(obj)
-  });
+  var stand = {
+    obj: obj,
+    method: method
+  };
+  var find = internals.store.lookUp(obj, method);
+  Assert.equal(find.result, undefined, 'there is already a replace for "obj"[' + method + ']');
```
Comment (cjihrig) [inline lib/index.js]: Should this be `strictEqual()`?

File: README.md (hunk 2 above)
Comment (arb) [inline README.md:8]: Lets stick with semistandard @7.0.x until we have our own thing.

**Pull request merged**